salt/utils/process.py

```diff
@@ -147,7 +147,7 @@ def get_pidfile(pidfile):
             pid = pdf.read().strip()
         return int(pid)
     except (OSError, IOError, TypeError, ValueError):
-        return None
+        return int(-1)
```

Comment:
Just curious, isn't `int(-1)` and `-1` equivalent in Python? Why the extra `int()`?

Reply:
Removed int()